test/core/python/utility/pyboost_cv_converter.py.cpp

```diff
@@ -16,6 +16,21 @@
 #include <algorithm>
 #include <vector>
 
+// point //
+
+// check c++ -> python
+cv::Point2f point()
+{
+    cv::Point2f pt = cv::Point2f(4.2, 42);
+    return pt;
+}
+
+// check python -> c++
+bool check_point(cv::Point2f pt)
+{
+    return (pt.x - 4.2) < 0.0001 && (pt.y - 42.0) < 0.0001;
```

Comment:
This is a one-sided check, I guess you wanted to use `std::abs`.

Reply:
true, thanks